src/main.lua

```diff
@@ -370,6 +370,19 @@ end, function(msg)
 		lastKnownRequestSupply = LastKnownPnpRequestSupply,
 		lastKnownTotalSupply = lastKnownTotalTokenSupply(),
 	}
+
+	-- Format all incoming addresses
+	msg.From = utils.formatAddress(msg.From)
+	msg.Tags.Recipient = msg.Tags.Recipient and utils.formatAddress(msg.Tags.Recipient) or nil
+	msg.Tags.Initiator = msg.Tags.Initiator and utils.formatAddress(msg.Tags.Initiator) or nil
+	msg.Tags.Target = msg.Tags.Target and utils.formatAddress(msg.Tags.Target) or nil
+	msg.Tags.Source = msg.Tags.Source and utils.formatAddress(msg.Tags.Source) or nil
+	msg.Tags.Address = msg.Tags.Address and utils.formatAddress(msg.Tags.Address) or nil
+	msg.Tags["Vault-Id"] = msg.Tags["Vault-Id"] and utils.formatAddress(msg.Tags["Vault-Id"]) or nil
+	msg.Tags["Process-Id"] = msg.Tags["Process-Id"] and utils.formatAddress(msg.Tags["Process-Id"]) or nil
+	msg.Tags["Observer-Address"] = msg.Tags["Observer-Address"] and utils.formatAddress(msg.Tags["Observer-Address"])
+		or nil
+
```

Comment:
nit add Quantity and convert it to number

Reply:
[119a06f](https://github.com/ar-io/ar-io-network-process/pull/183/commits/119a06f78258d77ff51bbf0e5f9e62360c94d207)